SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/WorkerCoordinator/ManagedWorkerCoordinator.cs

```diff
@@ -163,42 +162,28 @@ private void StartSimulatedPlayer(string simulatedPlayerName, Option<string> dev
         {
             try
             {
-                // Create player identity token and login token
-                string pit = "";
-                string loginToken = "";
-                if (devAuthTokenOpt.HasValue)
+                // Pass in the dev auth token and the target deployment
+                if (devAuthTokenOpt.HasValue && targetDeploymentOpt.HasValue)
                 {
-                    pit = Authentication.GetDevelopmentPlayerIdentityToken(devAuthTokenOpt.Value, simulatedPlayerName);
-
-                    if (targetDeploymentOpt.HasValue)
-                    {
-                        var loginTokens = Authentication.GetDevelopmentLoginTokens(SimulatedPlayerWorkerType, pit);
-                        loginToken = Authentication.SelectLoginToken(loginTokens, targetDeploymentOpt.Value);
-                    }
-                    else
-                    {
-                        Logger.WriteLog($"Not generating a login token for player {simulatedPlayerName}, no target deployment provided through worker flag \"{TargetDeploymentWorkerFlag}\".");
-                    }
+                    string[] simulatedPlayerArgs = Util.ReplacePlaceholderArgs(SimulatedPlayerArgs, new Dictionary<string, string>() {
+                        { SimulatedPlayerWorkerNamePlaceholderArg, simulatedPlayerName },
+                        { DevAuthTokenPlaceholderArg, devAuthTokenOpt.Value },
+                        { TargetDeploymentPlaceholderArg, targetDeploymentOpt.Value }
+                });
```

Comment:
One tab right? Maybe just GitHub messing up

Reply:
ah wait, I think visual studio messed this up a bit. Will check it again